nrf_security/Kconfig.legacy

```diff
@@ -301,7 +301,8 @@ config OBERON_BACKEND
 	bool
 	prompt "Configuration to enable nrf_oberon for legacy mbed TLS APIs"
 	select NRF_OBERON
-	depends on !CC3XX_BACKEND_FORCED && !CC3XX_BACKEND
+	depends on !CC3XX_BACKEND_FORCED
+		#  && !CC3XX_BACKEND - Can't add filter because of CI
```

Comment:
we are we adjusting code for CI, and not fixing CI in first place.

This code will go to users, CI don't go to users.